docs/quickstart_databricks.md

```diff
@@ -11,9 +11,32 @@ For Databricks, you can simply upload [this notebook](./samples/databricks/datab
 
 ![Import Notebooks](./images/databricks_quickstart1.png)
 
-
 2. Paste the [link to Databricks getting started notebook](./samples/databricks/databricks_quickstart_nyc_taxi_driver.ipynb):
 
 ![Import Notebooks](./images/databricks_quickstart2.png)
 
 3. Run the whole notebook. It will automatically install Feathr in your cluster and run the feature ingestion jobs.
+
+# Authoring Feathr jobs in local environment and submit to remote Databricks cluster
+
+Not everyone wants to use databricks notebook as the main development environment, and the above part is more for quick start purpose. For a more serious development, we usually recommend using Visual Studio Code, where [it has native support for Python and Jupyter Notebooks](https://code.visualstudio.com/docs/datascience/jupyter-notebooks) with many great features such as syntax highlight and IntelliSense.
```

Comment:
>Not everyone wants to use databricks notebook as the main development environment, and the above part is more for quick start purpose. For a more serious development, we usually recommend using Visual Studio Code [](http://example.com/codeflow?start=0&length=212)

Feel it's better to describe in this way: Although Databricks Notebooks are great tools, there are also larger developer communities that prefer the usage of VS Code.